src/components/courses/components/course-card/courses-cards.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, List} from "reactstrap";
+import toHoursAndMinutes from "../../../../common/to-hours-and-minutes";
+
+function CoursesCards({courses, authors}) {
+
+	function authorsNameById(authorsId) {
+		return authorsId.map(authorId => authors.find(({ id }) => id === authorId)?.name).join(', ')
+	}
+
+	return (
+		<List className='mt-1'>
+			{courses.map((course) => {
```

Comment:
Use `isEmpty` from lodash.isEmpty and optional chaining in case of operating data to prevent page breaking